File: AmortizationCalculator/Models/Payment.cs
public class Payment
{
    public int paymentID { get; set; }
    public DateTime LoanMonth { get; set; }
    public double MonthlyPayment { get; set; } = 0;
    public double AmountLeft { get; set; }
    public double Principal { get; set; }
    public double Interest { get; set; }
    public int LoanID { get; set; }
}

File: FrontEnd/src/app/_models/payment.ts
export class Payment {
    loanMonth: string;
    interest: number;
    principal: number;
    amountLeft: number;
    monthlyPayment: number;
  
    constructor(loanMonth: string, interest: number, principal: number, amountLeft: number, monthlyPayment: number) {
      this.loanMonth = loanMonth;
      this.interest = interest;
      this.principal = principal;
      this.amountLeft = amountLeft;
      this.monthlyPayment = monthlyPayment;
    }
  }
  